subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -114,10 +114,15 @@ static int download_client_callback(const struct download_client_evt *event)
 			download_client_disconnect(&dlc);
 			LOG_ERR("Download client error");
 			err = dfu_target_done(false);
-			if (err != 0) {
+			if (err != 0 && err != -EACCES) {
 				LOG_ERR("Unable to deinitialze resources "
 					"used by dfu_target.");
 			}
+
+			if (IS_ENABLED(CONFIG_FOTA_DOWNLOAD_LOG_LEVEL_DBG)
```

Comment:
I don't see a need for this condition here, the log level is already covered within `LOG_DBG`.

Reply:
Ah okei. I'll fix it the way you said thanks :)

Reply:
Fixed.